code/modules/vore/eating/living_vr.dm

```diff
@@ -21,7 +21,8 @@
 	var/fuzzy = 1						// Preference toggle for sharp/fuzzy icon.
 	var/tail_alt = 0					// Tail layer toggle.
 	var/can_be_drop_prey = 0
-	var/can_be_drop_pred = 1	//Mobs are pred by default.
+	var/can_be_drop_pred = 1			// Mobs are pred by default.
+	var/fruit_type = "apple"			// Not used outside of Alraunes. This is on all /living just in case some other mob gets it or it's being bussed in an event. Mobs with fruit, anyone?
```

Comment:
## n o p e

No, don't add this to every living mob in the entire game for a single species which, most of the time, nobody will be playing.

Reply:
Where should I put the variable? I just put it here since this is where all the other vore variables are, just like xenochimera's "feral" variable that's here.

Reply:
Ah, wait, nevermind. Found where I need to put it https://github.com/VOREStation/VOREStation/blob/master/code/modules/mob/living/carbon/human/human_defines_vr.dm